lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -93,7 +93,6 @@
                             });
                         }
                     );
-                    $('.wrapper-prompt').focus();
```

Comment:
The 'leave team' confirmation prompt uses the same js.  I removed this `.focus()` in favor of the new behavior.